app/javascript/packs/styles/main.scss

```diff
@@ -1,8 +1,70 @@
 @import 'constants';
+@import '~react-bootstrap-table/dist/react-bootstrap-table.min';
 
 body {
   background-color: $color-concrete;
   color: $color-emperor;
   font-family: "Open Sans",Arial,Helvetica,sans-serif;
   font-size: 13px;
 }
+
+.react-bs-table {
+  thead {
+    background-color: #909088;
```

Comment:
please use constants.scss if you style in components with styled-components use app/constants/colors